src/index.test.ts

```diff
@@ -1,9 +1,23 @@
-import { encryptString, decryptString, ENCRYPTION_PARAMETERS, IEncryptionResults } from ".";
+import {
+  encryptString,
+  decryptString,
+  ENCRYPTION_PARAMETERS,
+  IEncryptionResults,
+  encodeDocument,
+  decodeDocument
+} from ".";
 
 const base64Regex = /^(?:[a-zA-Z0-9+/]{4})*(?:|(?:[a-zA-Z0-9+/]{3}=)|(?:[a-zA-Z0-9+/]{2}==)|(?:[a-zA-Z0-9+/]{1}===))$/;
 const encryptionKeyRegex = new RegExp(`^[0-9a-f]{${ENCRYPTION_PARAMETERS.keyLength / 4}}$`);
 
 describe("storage/crypto", () => {
+  it("should encrypt and decrypt unicode symbols correctly", () => {
+    const originalObject = JSON.stringify({ data: "Rating(≤ 25kg)" });
```

Comment:
Do you want to test with more chars like for `encodeDocument & decodeDocument`

Reply:
Added a separate test for that